app/views/observations/index.html.erb

```diff
@@ -36,7 +36,7 @@
 <% end %>
 
 <%= paginate_block(@pages) do %>
-  <div class="row results mt-3">
+  <div class="row mt-3">
     <%= render(partial: "shared/matrix_box", layout: "shared/matrix_table",
```

Comment:
This was always a superfluous class here. It's rendered in the front end as basically:
```
<div class="results">
  <br>
  <ul class="pagination pagination-sm">
  <div class="row results mt-3">
```